tools-local/prebuilt-baseline-offline.xml

```diff
@@ -136,18 +138,17 @@
     <Usage Id="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <Usage Id="Microsoft.NET.Test.Sdk" Version="15.6.0" />
     <Usage Id="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+    <Usage Id="Microsoft.NETCore" Version="5.0.0" />
```

Comment:
Interesting... reeeally old package. Only usage `<AnnotatedUsage Id="Microsoft.NETCore" Version="5.0.0" File="src/vstest/src/Microsoft.TestPlatform.PlatformAbstractions/obj/project.assets.json" Project="src/vstest/" />
`

Reply:
It looks like including uap10.0 as a TargetFramework is pulling this in.  Maybe we should be conditionally removing this for source-build.